pkg/controller/direct/cloudbuild/externalresource.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudbuild
+
+import (
+	"strings"
+
+	cloudbuildpb "cloud.google.com/go/cloudbuild/apiv1/v2/cloudbuildpb"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/controller/direct/externalresource"
+)
+
+func NewResourceRef(gcpObj *cloudbuildpb.WorkerPool) *ResourceRef {
+	return &ResourceRef{
+		externalresource.New("https://cloudbuild.googleapis.com/v1/", gcpObj),
```

Comment:
Nit: I don't know if we want v1.  Although I guess it depends on whether in OnePlatform a v1 and v2 with the same name would be the same resource.

Reply:
I see what you mean. I think we need to specify a version in the URL anyways to make it valid. Maybe I'm wrong, but does the v1/v2 bundles in the godoc.org/cloud.google.com/go?